crates/oxc_linter/src/snapshots/jsx_no_useless_fragment.snap

```diff
@@ -0,0 +1,166 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: jsx_no_useless_fragment
+---
+  ⚠ eslint-plugin-react(jsx-no-useless-fragment): Fragments should contain more than one child - otherwise, there's no need for a Fragment at all.
```

Comment:
Shall we move the "otherwise" part into the help section? The help should say why it's unnecessary.